common/src/main/java/com/wynntils/models/items/encoding/impl/block/PowderDataTransformer.java

```diff
@@ -159,7 +159,10 @@ private ErrorOr<PowderData> decodePowderData(ArrayReader<UnsignedByte> byteReade
             }
 
             // Add the powder to the data
-            data.add(new Pair<>(Powder.fromElement(Element.fromEncodingId(element)), tier));
+            Powder powder = Powder.fromElement(Element.fromEncodingId(element));
+            if (powder != null) { // Sometimes null when users mess with custom encoding
+                data.add(new Pair<>(powder, tier));
+            }
```

Comment:
Wouldn't it be smarter to just straight up error? If the powder info is not the last block, the whole decoding could be in a bad state from now on.